src/Slider.tsx

```diff
@@ -95,49 +95,43 @@ export const Slider = forwardRef<SliderTypes.API, PropsWithChildren<Settings>>((
     const mouseUpHandler = () => setIsDragging(false);
 
     const mouseDownHandler = (event: ReactMouseEvent<HTMLDivElement>) => {
-        setMousePosition({
+        mousePosition.current = {
```

Comment:
We use an imperative handle to return the API of the Slider instead of 'just' the ref (https://github.com/YourSurpriseCom/slider/blob/e64748bd54e9e6cba65f53c923d00443f508f906/src/Slider.tsx#L321C13-L321C13). I don't think using it is applicable here as this data is never exposed. I just want to store some data without causing a re-render if the data changes.